qfieldsync/gui/checker_feedback_table.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+"""
+/***************************************************************************
+ QFieldSync
+                                 A QGIS plugin
+ Sync your projects to QField on android
```

Comment:
```suggestion
 Sync your projects to QField
```

QField does more than Android !

Reply:
Fixed also in https://github.com/opengisch/qfieldsync/pull/531